### DISCUSSION TITLE
Have been trying for months.. M1 Max, nothing works

### DISCUSSION THREAD

**Discussion by rbutler81** [utmapp/UTM#4151] [General]
Hi there, hoping someone can lend a hand..

I've been trying on and off for several months to install some flavour of Linux with zero results.
I've tried Ubuntu Server ARM, and Kali mostly.

I can get to the Ubuntu ARM installer, the OS seems like it installs properly, but then I can never boot, and/or can't get a desktop GUI installed.

What I mean is, if I change the graphic setting to "console only" I can boot up into Ubuntu Server, but I can't install / run a desktop.
If I choose "full graphics" and then choose any emulated graphics card, nothing works.. in multiple ways:
- sometimes the VM just won't boot - screen stays black
- sometimes i can get to the boot menu but then UTM promptly crashes
- various error messages during boot which freeze the system

Has anyone been able to successfully run any ARM Linux distros on a Macbook Pro with M1 Max?
I'd love to hear what works here, as it's getting increasingly frustrating.
I can provide more details based on which graphics card is chosen if anyone has any ideas.

Thanks in advance.

**Comment by PhotoNomad0** [comment]
I did succeed with M1 Pro, but don't have an M1 Max to test.  Did you try using these instructions? https://mac.getutm.app/gallery/ubuntu-20-04

**Comment by rbutler81** [comment]
Hey, thanks for the reply.
Yes, I’ve followed those instructions… they didn’t work.
:/

On Sun, Jul 3, 2022 at 4:01 PM Bruce McLean ***@***.***>
wrote:

> I did succeed with M1 Pro, but don't have an M1 Max to test. Did you try
> using these instructions? https://mac.getutm.app/gallery/ubuntu-20-04
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/utmapp/UTM/discussions/4151#discussioncomment-3074245>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AHA6BRSCB4UU7NADLL4FXXDVSH5Q5ANCNFSM5ZKW67CQ>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
-- 

Ryan Butler
***@***.***
